test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -13,7 +14,11 @@ export default class PossibleIssues extends Component {
 
     async componentDidMount() {
         await this.fetchIssues();
-    }
+    };
+
+    getUserFeedback(repo_name, build_name, issue_name, issue_creator) {
```

Comment:
For this function, you can add a parameter (e.g. accuracy), which will be true for `smile`, and false for `frown`.
In addition, the naming convention should be camel case for all parameters.